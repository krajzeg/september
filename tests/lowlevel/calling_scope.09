print("On top level, the calling scope should be Nothing: ", vm.callerScope())

inject := {
	caller_scope := vm.callerScope()
	caller_scope::theAnswer = 42
}

inject()
print("This should be 42 after inject() sets it:", theAnswer);
