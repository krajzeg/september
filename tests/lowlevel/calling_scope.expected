On top level, the calling scope should be Nothing:  <Nothing>
This should be 42 after inject() sets it: 42
